src/Symfony/Component/JsonStreamer/DependencyInjection/StreamablePass.php

```diff
@@ -31,11 +31,11 @@ public function process(ContainerBuilder $container): void
 
         // retrieve concrete services tagged with "json_streamer.streamable" tag
         foreach ($container->getDefinitions() as $id => $definition) {
-            if (!$tag = ($definition->getTag('json_streamer.streamable')[0] ?? null)) {
+            if (!$tag = $definition->getTag('json_streamer.streamable')[0] ?? null) {
                 continue;
             }
 
-            if (($className = $container->getDefinition($id)->getClass()) && !$container->getDefinition($id)->isAbstract()) {
```

Comment:
the call to `isAbstract()` isn't bound to any particular meaning (at least none I'm aware of)